resources/admin/classes/domain/User.php

```diff
@@ -70,7 +69,7 @@ public function isAdmin(){
 	public function canEdit(){
 		$privilege = new Privilege(new NamedArguments(array('primaryKey' => $this->privilegeID)));
 
-		if ((strtoupper($privilege->shortName) == 'ADD/EDIT') || (strtoupper($privilege->shortName) == 'ADMIN')){
+		if ((mb_strtoupper($privilege->shortName) == 'ADD/EDIT') || (strtoupper($privilege->shortName) == 'ADMIN')){
```

Comment:
The second `strtoupper` didn't get changed. When it is also `mb_strtoupper` the issue is fixed on my test. 
```suggestion
		if ((mb_strtoupper($privilege->shortName) == 'ADD/EDIT') || (mb_strtoupper($privilege->shortName) == 'ADMIN')){
```